backend/requirements.txt

```diff
@@ -1,2 +1,4 @@
 fastapi[all]
 uvicorn
+git+https://github.com/openai/whisper.git
```

Comment:
Does it actually work when I do `pip install -r requirements.txt`? you've said you had some guide written somewhere